File: app/src/main/java/kr/com/misemung/realm/entity/CityRecord.java
package kr.com.misemung.realm.entity;

import android.support.annotation.NonNull;

import io.realm.RealmModel;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.RealmClass;
import kr.com.misemung.vo.AirInfo;
import kr.com.misemung.vo.CityInfo;

@RealmClass
public class CityRecord implements RealmModel {

    @PrimaryKey
    public String umdName;
    public String tmX;
    public String tmY;


    public CityRecord() {}

    public CityRecord(@NonNull CityInfo cityInfo) {
        this.tmX = cityInfo.getTmX();
        this.tmY = cityInfo.getTmY();
    }

}

File: app/src/main/java/kr/com/misemung/realm/entity/CityRecord.kt
package kr.com.misemung.realm.entity

import io.realm.RealmModel
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass
import kr.com.misemung.vo.CityInfo

open class CityRecord : RealmObject {
    @PrimaryKey
    var id = 0
    var umdName: String? = null
    var tmX: String? = null
    var tmY: String? = null

    constructor() {}
    constructor(cityInfo: CityInfo) {
        tmX = cityInfo.tmX
        tmY = cityInfo.tmY
    }
}